### PR TITLE
Fix #560: Create README.md [ON HOLD]

### LINKED ISSUE #560: Add README.md
This is a part of documentation needed to bring new contributors onboard.
Mainly this should align with https://github.com/oppia/oppia/blob/develop/.github/README.md file.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

This PR focuses on documentation

Major files introduced or changed:

https://github.com/oppia/oppia-android/blob/rt4914-patch-1/.github/README.md
https://github.com/oppia/oppia-android/wiki#installation
https://github.com/oppia/oppia-android/wiki/Fork-and-Clone-Oppia-Android

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
@veena14cs @nikitamarysolomanpvt PTAL at this documentation. Once you guys approve this, I will pass this to @BenHenning and @seanlip for further improvements.
While reviewing this, think from the perspective of an android developer who wants to contribute towards open-source but has no idea what Oppia is.

**Comment by rt4914** [comment]
# [Oppia-Android](https://www.oppia.org)

Oppia-Android is a mobile application version of [Oppia](https://www.oppia.org). It is an Android application that enables anyone to easily learn through interactive activities (called 'explorations'). These activities simulate a one-on-one conversation with a tutor, making it possible for students to learn by doing while getting feedback.

Moreover, users can create their profile, practice a set of skills, play with explorations, review previously finished skills and also track their progress.

Oppia-Android app is written in Kotlin and Python, and is built on top of Google App Engine.

  * [Oppia.org community site](https://www.oppia.org)
  * [Contributors' wiki](https://github.com/oppia/oppia-android/wiki)
  * [Developer mailing list](http://groups.google.com/group/oppia-android-dev)
  * [File an issue](https://github.com/oppia/oppia-android/issues/new/)

<p align="center">
  Screenshot 2019-12-18 at 4 37 57 PM
</p>


## Installation

Please refer to the [developer wiki](https://github.com/oppia/oppia-android/wiki) for full installation instructions.

## Contributing

The Oppia-Android project is built by the community for the community. We welcome contributions from everyone, especially new contributors.

If you'd like to help out, please check out [this wiki page](https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change).

## Support

If you have any feature requests or bug reports, please log them on our [issue tracker](https://github.com/oppia/oppia-android/issues/new).

Please report security issues directly to admin@oppia.org.


## License

The Oppia-Android code is released under the [Apache v2 license](https://github.com/oppia/oppia-android/blob/develop/LICENSE).


## Keeping in touch

  * [Blog](https://medium.com/oppia-org)
  * Social media: [YouTube](https://www.youtube.com/channel/UC5c1G7BNDCfv1rczcBp9FPw), [FB](https://www.facebook.com/oppiaorg), [Twitter](https://twitter.com/oppiaorg)

We also have public chat rooms on Gitter: [https://gitter.im/oppia/oppia-android](https://gitter.im/oppia/oppia-android). Drop by and say hello!

File: .github/README.md
Comment (veena14cs) [inline .github/README.md]: Does this link need access permission.

Comment (rt4914) [reply .github/README.md]: I believe yes. But I think these links needs to be verified by @BenHenning and @seanlip once I pass it to them

Comment (veena14cs) [reply .github/README.md]: Okay, I tried to open this link and it asked for permission or it said the group doesn't exist.

**Review by veena14cs — APPROVED** [review]
LGTM.

File: .github/README.md
Comment (nikitamarysolomanpvt) [inline .github/README.md]: Please correct spelling to "practice"

Comment (nikitamarysolomanpvt) [inline .github/README.md]: Correct spelling "explorations"

File: .github/README.md
Comment (nikitamarysolomanpvt) [inline .github/README.md]: It seems that you have an unnecessary comma in a compound predicate, and passive voice misuse.
"Oppia-Android app is written in Kotlin and python and built on top of Google App Engine."

File: .github/README.md
Comment (nikitamarysolomanpvt) [inline .github/README.md]: Please change to "Moreover, users can create their profile, practice a set of skills, play with explorations, review previously finished skills and also track the progress." issue in compound/complex sentence.

File: .github/README.md
Comment (nikitamarysolomanpvt) [inline .github/README.md]: Please change to "This is just a summary for developers who would like to contribute:"
short summary create a tautology

Comment (nikitamarysolomanpvt) [inline .github/README.md]: Please remove the word short before summary.
"This is just a short summary for developers who would like to contribute:"

File: .github/README.md
Comment (nikitamarysolomanpvt) [inline .github/README.md]: Nit : "It is an android application that enables anyone to learn easily interactive activities (called 'explorations'). Misplaced words or phrases" misplaced words/phrases

**Review by nikitamarysolomanpvt — COMMENTED** [review]
PTAL on comments

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Done.

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Done.

Comment (rt4914) [reply .github/README.md]: Done.

Comment (rt4914) [reply .github/README.md]: Done.

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Done.

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Removed short word.

Comment (rt4914) [reply .github/README.md]: Don.e

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Thanks.

File: .github/README.md
Comment (seanlip) [inline .github/README.md]: python --> Python

File: .github/README.md
Comment (seanlip) [inline .github/README.md]: android --> Android

learn easily --> easily learn through

File: .github/README.md
Comment (seanlip) [inline .github/README.md]: the progress --> their progress

File: .github/README.md
Comment (seanlip) [inline .github/README.md]: The link provided here does not exist. The folder name that is created is wrong.

File: .github/README.md
Comment (seanlip) [inline .github/README.md]: android studio --> Android Studio

File: .github/README.md
Comment (seanlip) [inline .github/README.md:25]: We should check with @BenHenning, before putting this up, whether the project is ready to accept new contributors.

File: .github/README.md
Comment (seanlip) [inline .github/README.md]: I don't think art, design or documentation are relevant for this project, at this stage. I think you can drop all this text and replace it with something simple like "If you'd like to help out, please check out this wiki page."

File: .github/README.md
Comment (seanlip) [inline .github/README.md]: This group does not exist. Please delete this line, for now.

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks for doing this. Please make sure there are no broken links.

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Updated.

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Updated.

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Updated

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Updated to `https://github.com/oppia/oppia/wiki/Fork-and-Clone-Oppia`

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Updated

File: .github/README.md
Comment (rt4914) [reply .github/README.md:25]: Ack.

File: .github/README.md
Comment (seanlip) [reply .github/README.md]: No, wrong link. That's not relevant to this repo.

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Updated.

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Removed.

**Comment by rt4914** [comment]
@seanlip Updated as per your suggestions.

File: .github/README.md
Comment (seanlip) [inline .github/README.md]: Please remove; this link is not relevant to this app.

**Review by seanlip — CHANGES_REQUESTED** [review]
Noticed one more thing. In addition, I left one other comment here: https://github.com/oppia/oppia-android/pull/564#discussion_r360313268

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Removed.

**Comment by seanlip** [comment]
@rt4914 Please look at the history and ensure that you have responded to all comments (including the one I linked) before assigning back to the reviewer. Thanks.

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: @seanlip Actually, we will need fork and clone in oppia-android too. So should we create a similar page in oppia-android too?

**Comment by seanlip** [comment]
Yes, please go ahead.

>

File: .github/README.md
Comment (rt4914) [reply .github/README.md]: Created `Fork and clone` page for `oppia-android`.

**Comment by rt4914** [comment]
@seanlip PTAL. I have added `fork-and-clone` page for `oppia-android` too which has also been linked to this PR.
https://github.com/oppia/oppia-android/wiki/Fork-and-Clone-Oppia-Android

**Comment by seanlip** [comment]
Thanks @rt4914. The PR looks good but I have one concern about the wiki page: "github help page" in the first sentence has no link. Could you add it?

Otherwise, no further comments from me. Thanks! (Let's wait for @BenHenning to give the go-ahead before merging this, though.)

**Comment by rt4914** [comment]
> Thanks @rt4914. The PR looks good but I have one concern about the wiki page: "github help page" in the first sentence has no link. Could you add it?
> 
> Otherwise, no further comments from me. Thanks!

Updated as per your comment. Now changing the title to [ON HOLD]

**Comment by rt4914** [comment]
Also, unassigning @BenHenning as this won't be on priority almost till January end.

**Comment by seanlip** [comment]
Actually, more notes for that page:

- The image in step 2 is quite blurry. Could you add a sharper one?
- Please add a link for "your repositories" in step 3.

I also made a few cosmetic edits to make it a bit easier to read.

Thanks!

**Comment by rt4914** [comment]
> Actually, more notes for that page:
> 
> * The image in step 2 is quite blurry. Could you add a sharper one?
> * Please add a link for "your repositories" in step 3.
> 
> I also made a few cosmetic edits to make it a bit easier to read.
> 
> Thanks!

1. Updated the image.
2. Which link should I add. `https://github.com/USERNAME?tab=repositories` this one? or something else.

**Comment by seanlip** [comment]
Hm, for (2), good point. Maybe just github.com (which shows the repos on the left)?

**Comment by rt4914** [comment]
> Hm, for (2), good point. Maybe just github.com (which shows the repos on the left)?

Done both.

**Comment by rt4914** [comment]
@seanlip as per our conversation on hangout, I have shifted the `Installation` section from `README.md` to `Home Wiki`.

Check this: Major files introduced or changed:

https://github.com/oppia/oppia-android/blob/rt4914-patch-1/.github/README.md
https://github.com/oppia/oppia-android/wiki/Fork-and-Clone-Oppia-Android

**Comment by seanlip** [comment]
Thanks! Looks great.

**Comment by rt4914** [comment]
@seanlip Should we now merge this PR, as we already have a lot of contributors at the same time we are accepting new contributors too.

**Comment by seanlip** [comment]
That's @BenHenning's call.

**Comment by BenHenning** [comment]
Since this hasn't seen progress in a while, closing it. We can still use it as a reference, or reopen if we want to continue contributing to this branch.

**Pull request closed without merging**
